cli-setup.sh

```diff
@@ -302,6 +302,11 @@ validate_platform() {
      stdout_log "Unsupported Linux distribution"
      exit 99
   fi
+
+  # check for locale
+  if [[ "${LANG}" != "en_US.UTF-8" ]]; then
+    stdout_log "Warning: Expected locale en_US.UTF-8 but found ${LANG}. CLI Installation may fail"
```

Comment:
It would need to be set to the locale that is supported by the system. So we may be breaking if if we blindly set it to `en_US.UTF-8` . We can have a write-up linking to resources that talk about system locale and unicode support as part of CLI docs. What do you think?

Reply:
Yes, I don't mean we set it. I meant more to suggest how one can set it if they want to do so. Either as text or a link to a doc in the CLI setup. May be ask the user for confirmation before they proceed.

Reply:
In the case that the CLI installation fails, the suggested locale settings for the system and how to set it is specified by the click error itself as below:
```
Traceback (most recent call last):
  File "/root/pf9/pf9-venv/bin/express", line 8, in <module>
    sys.exit(cli())
  File "/root/pf9/pf9-venv/lib/python3.5/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/root/pf9/pf9-venv/lib/python3.5/site-packages/click/core.py", line 760, in main
    _verify_python3_env()
  File "/root/pf9/pf9-venv/lib/python3.5/site-packages/click/_unicodefun.py", line 130, in _verify_python3_env
    " mitigation steps.{}".format(extra)
RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment. Consult https://click.palletsprojects.com/python3/ for mitigation steps.

This system supports the C.UTF-8 locale which is recommended. You might be able to resolve your issue by exporting the following environment variables:

    export LC_ALL=C.UTF-8
    export LANG=C.UTF-8

Click discovered that you exported a UTF-8 locale but the locale system could not pick up from it because it does not exist. The exported locale is 'en_US.UTF-8' but it is not supported
DEBUGGING: 20:27:46 : 63 : ASSERT: Base Installation of Platform9 CLI Failed
```
Do we just want to stick with addition of the warning alone? I plan to generalize this to all unicode locales instead of just `en_US`. Will update the review.

Reply:
That is in the log or the console? I would say let's get this change in and we can enhance the messaging after further discussion.